### ISSUE TITLE
Crash on start under FreeBSD

### ISSUE THREAD

**Issue by Thefrank** [bropat/ioBroker.eufy-security#106]
**Describe the bug**  
crash on start under FreeBSD 12.2

**To Reproduce**  
Install adapter from https://github.com/bropat/ioBroker.eufy-security
Configure user name and password
(install "web" adaptor)
Start  

**Expected behavior**  
It gathers data about my doorbell (it appears in objects) but crashes almost instantly after starting

**Screenshots & Logfiles**  
```

2021-04-01 11:52:08.204 - info: host.iobroker "system.adapter.eufy-security.0" enabled
--
2021-04-01 11:52:08.310 - info: host.iobroker instance system.adapter.eufy-security.0 started with pid 38913
2021-04-01 11:52:09.405 - info: web.0 (38912) starting. Version 3.3.0 in /opt/iobroker/node_modules/iobroker.web, node: v15.5.1, js-controller: 3.2.16
2021-04-01 11:52:09.635 - info: web.0 (38912) socket.io server listening on port 8082
2021-04-01 11:52:09.652 - info: web.0 (38912) Connect extension "eufy-security/build/lib/web.js"
2021-04-01 11:52:09.656 - info: web.0 (38912) http server listening on port 8082
2021-04-01 11:52:09.759 - debug: eufy-security.0 (38913) Redis Objects: Use Redis connection: 127.0.0.1:9001
2021-04-01 11:52:09.815 - debug: eufy-security.0 (38913) Objects client ready ... initialize now
2021-04-01 11:52:09.818 - debug: eufy-security.0 (38913) Objects create System PubSub Client
2021-04-01 11:52:09.819 - debug: eufy-security.0 (38913) Objects create User PubSub Client
2021-04-01 11:52:09.820 - debug: eufy-security.0 (38913) Objects client initialize lua scripts
2021-04-01 11:52:09.830 - debug: eufy-security.0 (38913) Objects connected to redis: 127.0.0.1:9001
2021-04-01 11:52:09.839 - debug: eufy-security.0 (38913) objectDB connected
2021-04-01 11:52:09.842 - debug: eufy-security.0 (38913) Redis States: Use Redis connection: 127.0.0.1:9000
2021-04-01 11:52:09.849 - debug: eufy-security.0 (38913) States create System PubSub Client
2021-04-01 11:52:09.850 - debug: eufy-security.0 (38913) States create User PubSub Client
2021-04-01 11:52:09.856 - debug: eufy-security.0 (38913) States connected to redis: 127.0.0.1:9000
2021-04-01 11:52:09.857 - debug: eufy-security.0 (38913) statesDB connected
2021-04-01 11:52:09.983 - info: eufy-security.0 (38913) starting. Version 0.5.1 in /opt/iobroker/node_modules/iobroker.eufy-security, node: v15.5.1, js-controller: 3.2.16
2021-04-01 11:52:10.096 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.111 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.117 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.122 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.128 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.133 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.138 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.144 - debug: eufy-security.0 (38913) No suitable Lua script, fallback to keys!: function(doc) { if (doc.type === 'state') emit(doc._id, doc) }
2021-04-01 11:52:10.151 - error: eufy-security.0 (38913) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2021-04-01 11:52:10.151 - error: eufy-security.0 (38913) unhandled promise rejection: Cannot read property 'val' of null
2021-04-01 11:52:10.153 - error: eufy-security.0 (38913) TypeError: Cannot read property 'val' of null
at EufySecurity. (/opt/iobroker/node_modules/iobroker.eufy-security/build/main.js:317:27)
at Generator.next ()
at fulfilled (/opt/iobroker/node_modules/iobroker.eufy-security/build/main.js:27:58)
2021-04-01 11:52:10.153 - error: eufy-security.0 (38913) Cannot read property 'val' of null
2021-04-01 11:52:10.202 - info: eufy-security.0 (38913) terminating
2021-04-01 11:52:10.203 - warn: eufy-security.0 (38913) Terminated (UNCAUGHT_EXCEPTION): Without reason
2021-04-01 11:52:10.831 - error: host.iobroker instance system.adapter.eufy-security.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
```

**Versions:**  
 - Adapter version: 0.5.1
 - JS-Controller version: 3.2.16
 - Node version: v15.5.1
 - Operating system: FreeBSD 12.2

**Additional context**  
many node packages don't build/work for FreeBSD so even getting this installed was a challenge (e.g., had to manually build FFMPEG and get it added to `ffmpeg-static`)

**Comment by Thefrank** [comment]
I deleted and reinstalled the adaptor. now I am getting
```

2021-04-01 21:00:11.114 - debug: eufy-security.0 (2249) Station.connect(): station: T8200N------ p2p_did: SECCAMA-203575-- dsk_key: -----------------------
--
2021-04-01 21:00:12.178 - error: eufy-security.0 (2249) uncaught exception: Could not get or set buffer size: uv_recv_buffer_size returned ENOBUFS (no buffer space available)
2021-04-01 21:00:12.178 - error: eufy-security.0 (2249) SystemError [ERR_SOCKET_BUFFER_SIZE]: Could not get or set buffer size: uv_recv_buffer_size returned ENOBUFS (no buffer space available)
at new SystemError (node:internal/errors:217:5)
at new NodeError (node:internal/errors:302:7)
at bufferSize (node:dgram:196:11)
at Socket.setRecvBufferSize (node:dgram:955:3)
at Socket. (/opt/iobroker/node_modules/eufy-security-client/build/p2p/session.js:231:37)
at Socket.onListening (node:dgram:243:7)
at Socket.emit (node:events:378:20)
at startListening (node:dgram:168:10)
at node:dgram:363:7
at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**Comment by bropat** [comment]
Hi @Thefrank ,

this adapter does not actually support FreeBSD.
But I will see what I can do.

**Comment by bropat** [comment]
Try the new version (0.5.2) and let me know.

**Comment by Thefrank** [comment]
The issue is resolved with 0.5.2

Thank you!
